src/main/java/games/strategy/engine/lobby/server/db/UserDao.java

```diff
@@ -4,15 +4,17 @@
 
 public interface UserDao {
 
-  HashedPassword getPassword(String userName);
+  HashedPassword getPassword(String username);
```

Comment:
Looks like we have almost 50/50 `username` vs `userName`

```
dan@dan-desk:~/work/triplea$ find . -type f -name "*java" | xargs grep -o "username" | wc -l
162
dan@dan-desk:~/work/triplea$ find . -type f -name "*java" | xargs grep -o "userName" | wc -l
163
```

This update would tip the balance towards `username`. I think the existing camel case was actually more 'correct'. User name is two words, an adjective plus a noun. Username is not a word. So the camel case of two words would be `userName`

Reply:
I disagree. While 'User' and 'Name' can indeed be written separately, they do not make sense in this order.
'HashedPassword' however would make sense written separately IMO, but that's just me